protocol/invocation/rpcinvocation.go

```diff
@@ -69,6 +69,14 @@ func NewRPCInvocationForProvider(methodName string, arguments []interface{}, att
 	}
 }
 
+func NewRPCInvocationForUT(methodName string, parameterTypes []reflect.Type, arguments []interface{}) *RPCInvocation {
```

Comment:
It's not good to write somethis about ut in formal code. Why do you use existing method? And can you change it to the following？
![image](https://user-images.githubusercontent.com/18375310/60384685-29a4cb00-9ab3-11e9-9b66-45fecf83779b.png)